<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crop Images</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CropGraphicsView" name="graphicsView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
      <widget class="QFrame" name="framePreview">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImagePreview" name="labelPreview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFiles"/>
   <addaction name="actionOpenDir"/>
   <addaction name="separator"/>
   <addaction name="actionDestDir"/>
   <addaction name="separator"/>
   <addaction name="actionPreviousFile"/>
   <addaction name="actionNextFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionCropImage"/>
   <addaction name="actionSaveFile"/>
   <addaction name="actionRestoreImage"/>
   <addaction name="actionFadeMask"/>
   <addaction name="separator"/>
   <addaction name="actionFit"/>
   <addaction name="actionScaleM10"/>
   <addaction name="actionScaleP10"/>
   <addaction name="separator"/>
   <addaction name="actionRotL"/>
   <addaction name="actionRotR"/>
   <addaction name="separator"/>
   <addaction name="actionMirrorH"/>
   <addaction name="actionMirrorV"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionCropImage">
   <property name="text">
    <string>Crop</string>
   </property>
   <property name="toolTip">
    <string>Crop image</string>
   </property>
  </action>
  <action name="actionFadeMask">
   <property name="text">
    <string>Fade</string>
   </property>
   <property name="toolTip">
    <string>Set cropping fade</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="text">
    <string>Fit</string>
   </property>
   <property name="toolTip">
    <string>Fit picture in window</string>
   </property>
  </action>
  <action name="actionScaleP10">
   <property name="text">
    <string>+10%</string>
   </property>
   <property name="toolTip">
    <string>Scale +10%</string>
   </property>
  </action>
  <action name="actionScaleM10">
   <property name="text">
    <string>-10%</string>
   </property>
   <property name="toolTip">
    <string>Scale -10%</string>
   </property>
  </action>
  <action name="actionRotR">
   <property name="text">
    <string>+90</string>
   </property>
   <property name="toolTip">
    <string>Rotate image right</string>
   </property>
  </action>
  <action name="actionRotL">
   <property name="text">
    <string>-90</string>
   </property>
   <property name="toolTip">
    <string>Rotate image left</string>
   </property>
  </action>
  <action name="actionMirrorV">
   <property name="text">
    <string>--</string>
   </property>
   <property name="toolTip">
    <string>Mirror vertical</string>
   </property>
  </action>
  <action name="actionMirrorH">
   <property name="text">
    <string>|</string>
   </property>
   <property name="toolTip">
    <string>Mirror horizontal</string>
   </property>
  </action>
  <action name="actionRestoreImage">
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore original image</string>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="text">
    <string>OpenFiles</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>OpenDir</string>
   </property>
  </action>
  <action name="actionPreviousFile">
   <property name="text">
    <string>PreviousFile</string>
   </property>
  </action>
  <action name="actionNextFile">
   <property name="text">
    <string>NextFile</string>
   </property>
  </action>
  <action name="actionDestDir">
   <property name="text">
    <string>DestDir</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CropGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>src/CropGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>ImagePreview</class>
   <extends>QLabel</extends>
   <header>src/ImagePreview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
